src/common/opencl.h

```diff
@@ -403,7 +403,7 @@ cl_ulong dt_opencl_get_device_available(const int devid);
 cl_ulong dt_opencl_get_device_memalloc(const int devid);
 
 /** round size to a multiple of the value given in the device specifig config parameter for opencl_size_roundup */
-int dt_opencl_dev_roundup(int size, const int devid);
```

Comment:
for clarity I'll use an enum here instead of int and use that in the implementation.

Reply:
maybe even better to use two straight functions like `dt_opencl_dev_roundup_width()`

Reply:
Would works for me too. As you prefer.